packages/tree-widget/src/components/CategoriesTree.scss

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@bentley/ui-core/lib/ui-core/index";
+
+.ui-test-app-visibility-widget {
+  padding: $uicore-s;
+  width: 100%;
+  height: 100%;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  .components-selectable-content {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    width: 100%;
+  }
+
+  .components-selectable-content-header {
+    display: flex;
+    align-items: center;
+    margin-bottom: $uicore-xs;
+
+    .components-selectable-content-selector {
+      flex: 1;
+      padding: 5px;
+      border: 1px solid #ccc;
+      width: 50%;
+      margin: 0;
+    }
+  }
+
+  .components-selectable-content-wrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .ui-test-app-visibility-tree-toolbar {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: $uicore-xs;
+
+    .tree-toolbar-searchbox {
+      margin-left: auto;
+
+      .components-filtering-input {
+        margin: 0;
+      }
+    }
+  }
+}
+
+.ui-test-app-select {
+  margin-bottom: $uicore-xs;
+}
+
+.ui-test-app-tree-container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+}
+
+.ui-test-app-search-bar {
+  margin-bottom: $uicore-s;
+}
```

Comment:
Can we get more descriptive & unique class names, **ui-test-app-xxx** should not be used.  Probably my bad from copy/paste.

Reply:
@aruniverse has also brought up removing `ui-test-app` (as well as `uisnip-core`). Bringing in more descriptive class names wouldn't hurt either.

Removed all traces of `ui-test-app` and `uisnip-core` prefixes and also adjusted class names to be more unique, descriptive and hopefully more clear.